crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -543,6 +543,12 @@ impl<'w> EntityMut<'w> {
         self.world
     }
 
+    /// Gives mutable access to this `EntityMut`'s [`World`] in a temporary scope.
```

Comment:
Might benefit from explaining why you'd want to use it.

Reply:
It's the only way of safely getting a `&mut World` from an `EntityMut` without consuming `self`. Which sort of speaks for itself? 
I'm not sure what to add to the docs that could make that clearer (Aside from being very pedantic) :)

Reply:
You could discuss update_location here, but that's very gory-internals, and likely to go out of date in dangerous ways. If people care, they can read the code.

Reply:
I was thinking: someone reading the doc doesn't necessarily see the source code. If they encounter this method, they may ask themselves why its here. But I agree with alice, it's really a minor thing and really depends on internals.